pallets/pools/src/lib.rs

```diff
@@ -1027,7 +1027,7 @@ pub mod pallet {
 				let new_solution = Self::score_solution(&pool, &epoch, &solution)?;
 				if let Some(ref previous_solution) = epoch.best_submission {
 					ensure!(
-						&new_solution > previous_solution,
+						&new_solution >= previous_solution,
```

Comment:
I wonder about one thing: if there are multiple solutions with the same score for an epoch, this makes it possible to keep overriding the best solution. I'm not sure if that's an issue. I don't think so, but maybe worth a little bit of thought.

A potential solution would be to only allow `>=` once.

Reply:
Never mind, I don't think this is an issue.